### PR TITLE
Athens docs

### PR DESCRIPTION
**Pull request by tangjeff0**
Does anyone know how to create markdown links with evil characters like `(`? 😂

https://github.com/athensresearch/athens/pull/82/files#diff-9e8744ec73212c1966bb19fa22fc3e0aR5-R11

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+1. [Roles and Responsibilities](#roles-and-responsibilities)
+   1. [Benevolent Dictator (project lead)](#benevolent-dictator)
+   1. [Core Team](#core-team)
+   1. [Moderators (Guardians)]([1])
+   1. [Contributors (Athenians)]([2])
+   1. [Other Roles](#other-roles)
+1. [Attribution](#attribution)
+
+[1]: [#moderators-(guardians)]
+[2]: [#contributors-(athenians)]
```
Comment (tangjeff0) [inline GOVERNANCE.md:11]: Does anyone know how to create markdown links with evil characters like `(`? 😂

**Comment by roryokane** [comment]
> Does anyone know how to create markdown links with evil characters like `(`?

Enclose the parenthesized URL within angle brackets `<>`:

~~~markdown
[test](<https://en.wikipedia.org/wiki/Test_(Unix)>)
~~~

> [test](<https://en.wikipedia.org/wiki/Test_(Unix)>)

I learned this trick from seeing Prettier [format my Markdown files][run-prettier] in projects where I was using it to format JavaScript. In general, I suggest running Markdown files through Prettier before publishing them to normalize their syntax.

[run-prettier]: https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBteBnGBdAFACxhgAcMkB6chAOgHcBLAa3uLgBN6BDaiAJwHNyDZuQAqcLAH1cAVSj0AHgEolIADQgIxGPWgZkoTr14RaABSMJ9KTgBtanAJ76NAI16cwjODADKxT3oofmQYXgBXOA1CAFtbAHV8ekwAsDhfK2T6ADdkx2RwDBcQIIw4XhgzD34YzmQAMzsyjQArDAUAIQ8vH19OGLgAGSC4BqaokDaFXyD+WzgARXCIeDHbZpAA3jLeAtreRjZTKHVN3iCYePo2GHxkAA4ABg1iEzL4j2IC14ly7NGNABHZbwKpaawgTgYAC0UDg7HYp14cGB9GRVU4NTqSEa6wmZRi9FCEXxs3mSxWoxx4w0ME4riuNzuSAATLSPPRbLMAMIQGK1AoSACsp3CZVE9OsuI22UiAEkoGwEH4wOdtABBRW+GCOeZrMoAXwNQA

File: VISION.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+This means the Web needs bi-directionality within more apps. And then it needs bi-directionality between apps.
+
+## A Protocol for Bi-directionality
+
+The Web should've always have been bi-directional. The reason why it's not is because it's a security nightmare. Letting someone write to your website or server by simply linking to your domain necessarily opens you up to malicious exploits. There have been a [few unsuccessful attempts at notifications](https://en.wikipedia.org/wiki/Linkback) when someone links to your blog. Even these have led to [DDoS attacks](https://en.wikipedia.org/wiki/Pingback).
+
+We need a chat messenger like the one [Max Krieger](http://a9.io/glue-comic/) envisions. Chat is one of the clearest examples of diminishing returns on knowledge. If you don't check Slack for one day, you'll never catch up. Even if you do, good luck finding it again. Furthermore, aside from these diminishing returns on data, conversations are naturally networked and self-referential. But all we have right now is a ceasless tidal wave of messages represented as an append-only log. Managing it all with search, one level of threads, and channels is Sisyphean at best.
```
Comment (KasperZutterman) [inline VISION.md:68]:
```suggestion
We need a chat messenger like the one [Max Krieger](http://a9.io/glue-comic/) envisions. Chat is one of the clearest examples of diminishing returns on knowledge. If you don't check Slack for one day, you'll never catch up. Even if you do, good luck finding it again. Furthermore, aside from these diminishing returns on data, conversations are naturally networked and self-referential. But all we have right now is a ceaseless tidal wave of messages represented as an append-only log. Managing it all with search, one level of threads, and channels is Sisyphean at best.
```
"ceasless" -> "ceaseless"

File: VISION.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+That multiplayer, joinable knowledge graphs will follow.
+
+And that we need to start getting exponential returns on collective intelligence.
+
+# A Community for Collaboration and Learning
+
+Lastly, the final Athens vision is for everyone, one day, to learn how to learn anything.
+
+To do that, it is mission critical to create a space and culture where we can [work with one another and learn from one another](https://github.com/athensresearch/athens/blob/master/CODE_OF_CONDUCT.md).
```
Comment (KasperZutterman) [inline VISION.md:125]:
```suggestion
To do that, it is mission-critical to create a space and culture where we can [work with one another and learn from one another](https://github.com/athensresearch/athens/blob/master/CODE_OF_CONDUCT.md).
```

File: doc/ClojureFam.md [hunk 4]
```diff
@@ -46,9 +46,9 @@ So why take part in the ClojureFam program?
 
 Questions come and go. Libraries and side-projects get abandoned. The flux of people can make us lose track of the individuals.
```
Comment (KasperZutterman) [inline doc/ClojureFam.md:47]:
```suggestion
Questions come and go. Libraries and side-projects get abandoned. The flux of people can make us lose track of individuals.
```

File: VISION.md [hunk 5]
```diff
@@ -13,41 +13,41 @@ This document can be read alongside [Athens Vision Mindmap v2](https://www.notio
...
+This isn't about any specific company's ethics, either. This is about users putting their second brains — their entire lives — on a public cloud. In plain-text. To be summoned at will by the government ([Facebook Transparency](https://transparency.facebook.com/government-data-requests/country/US), [Google Transparency](https://transparencyreport.google.com/user-data/overview?hl=en)).
 
-A self-hosted Athens solves this privacy and security problem.
+A self-hosted, open-source Athens solves this privacy and security problem.
 
 # An Individual Memex
 
-The closest approximations we have to the brain are Roam and neural networks. Both of these are graphs with bi-directional data flows.
+The closest approximations we have to the brain are Tools for Networked Thought and neural networks. Both of these are graphs networks with bi-directional data flows.
```
Comment (KasperZutterman) [inline VISION.md:38]:
```suggestion
The closest approximations we have to the brain are Tools for Networked Thought and neural networks. Both of these are graph networks with bi-directional data flows.
```

Comment (tangjeff0) [reply VISION.md:38]: Really appreciate your hawk eyes @KasperZutterman

**Pull request merged**